scalafix-tests/unit/src/test/scala/scalafix/tests/cli/CliSemanticSuite.scala

```diff
@@ -257,4 +258,54 @@ class CliSemanticSuite extends BaseCliSuite {
     )
   }
 
+  test("on-demand with scalacOptions") {
+    // Todo: Ondemand compilation doesn't take scalacOptions or the reporter doesn't gather all warns in 2.13
+    // see https://github.com/scalacenter/scalafix/pull/1215
+    if (ScalaVersions.isScala213) true
```

Comment:
should we open a separate issue to track that?

Reply:
ok